app/src/test/java/org/mozilla/vrbrowser/GleanMetricsServiceTest.kt

```diff
@@ -0,0 +1,34 @@
+package org.mozilla.vrbrowser
+
+import androidx.test.core.app.ApplicationProvider
+import mozilla.components.service.glean.Glean.initialize
+import mozilla.components.service.glean.Glean.setUploadEnabled
+import mozilla.components.service.glean.config.Configuration
+import mozilla.components.service.glean.config.Configuration.Companion.DEFAULT_TELEMETRY_ENDPOINT
+import mozilla.components.service.glean.testing.GleanTestRule
+import org.junit.Assert.assertEquals
+import org.junit.Rule
+import org.junit.Test
+import org.mozilla.vrbrowser.GleanMetrics.*
+
+
+@get:Rule
+val gleanRule = GleanTestRule(ApplicationProvider.getApplicationContext())
+
+class GleanMetricsServiceTest {
```

Comment:
Note that you need to declare which testrunner this should be executed with. I.e. either `@RunWith(RobolectricTestRunner::class)` or `@RunWith(AndroidJUnit4::class)` just before this line.